src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -204,10 +208,24 @@ public void closeMore() {
   }
 
   public void addContent(String content) {
+    waitForElementByElement(textArea);
+    textArea.sendKeys(content);
+    PageObjectLogging.log("addContent", "content was added", true);
+  }
+
+  public void addContentInSourceMode(String content) {
+    waitForElementByElement(sourceModeTextArea);
     sourceModeTextArea.sendKeys(content);
     PageObjectLogging.log("addContent", "content was added", true);
   }
 
+  public String copyContent()
+  {
+    waitForElementByElement(textArea);
+    String content = textArea.getText();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "content". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)